`Scalar.value()` NullPointerException: Add Synchronization

Encountered `NullPointerException` due to unsynchronized `Scalar.value()` in a multithreaded environment. Review and synchronize all `Scalar` usages in the project to prevent this issue.

Stacktrace:

```java
[INFO] --- jeo:1.0-SNAPSHOT:disassemble (bytecode-to-eo) @ jeo-it ---
[INFO] Trying to load classes for bytecode verification from [/home/runner/work/jeo-maven-plugin/jeo-maven-plugin/target/its/bytecode-to-eo/target/classes, /home/runner/work/jeo-maven-plugin/jeo-maven-plugin/target/its/bytecode-to-eo/target/test-classes]
[INFO] Disassembling is started with mode 'short'
[INFO] Disassembling files from 'target/classes' to 'target/generated-sources/jeo-xmir'
[INFO] Disassembling 'target/classes/org/eolang/jeo/Application.class' (568b)
[INFO] Disassembling 'target/classes/org/eolang/jeo/WithoutPackage.class' (297b)
[INFO] Disassembling 'target/classes/foo/Foo.class' (373b)
[INFO] 67 attributes loaded from 58 stream(s) in 96ms, 67 saved, 652 ignored: ["Ant-Version", "Archiver-Version", "Automatic-Module-Name", "Bnd-LastModified", "Build-Jdk", "Build-Jdk-Spec", "Built-By", "Bundle-ActivationPolicy", "Bundle-Activator", "Bundle-ClassPath", "Bundle-Copyright", "Bundle-Description", "Bundle-DocURL", "Bundle-DocUrl", "Bundle-License", "Bundle-Localization", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-RequiredExecutionEnvironment", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Comment-DynamicImport", "Comment-Header", "Created-By", "DynamicImport-Package", "EO-Dob", "EO-Revision", "EO-Version", "Eclipse-BundleShape", "Eclipse-ExtensibleAPI", "Eclipse-LazyStart", "Eclipse-SourceReferences", "Eclipse-SystemBundle", "Export-Package", "Export-Service", "Implementation-Title", "Implementation-URL", "Implementation-Vendor", "Implementation-Vendor-Id", "Implementation-Version", "Import-Package", "Include-Resource", "JCabi-Build", "JCabi-Date", "JCabi-Version", "JEO-Dob", "JEO-Revision", "JEO-Title", "JEO-Vendor", "JEO-Version", "Lints-Version", "Main-Class", "Manifest-Version", "Module-Requires", "Multi-Release", "Originally-Created-By", "Project-Name", "Provide-Capability", "Require-Bundle", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version", "Tool", "X-Compile-Source-JDK", "X-Compile-Target-JDK"]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.585 s
[INFO] Finished at: 2025-03-28T07:18:12Z
[INFO] ------------------------------------------------------------------------
[WARNING] The requested profile "qulice" could not be activated because it does not exist.
[WARNING] The requested profile "long" could not be activated because it does not exist.
[ERROR] Failed to execute goal org.eolang:jeo-maven-plugin:1.0-SNAPSHOT:disassemble (bytecode-to-eo) on project jeo-it: Execution bytecode-to-eo of goal org.eolang:jeo-maven-plugin:1.0-SNAPSHOT:disassemble failed: java.lang.NullPointerException: Cannot invoke "java.lang.CharSequence.toString()" because the return value of "org.cactoos.Scalar.value()" is null -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException

```

Seems, the problem occurs in the `License`:

```java
[ERROR] Failed to execute goal org.eolang:jeo-maven-plugin:1.0-SNAPSHOT:disassemble (bytecode-to-eo) on project jeo-it: Execution bytecode-to-eo of goal org.eolang:jeo-maven-plugin:1.0-SNAPSHOT:disassemble failed: java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "java.lang.CharSequence.toString()" because the return value of "org.cactoos.Scalar.value()" is null -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.eolang:jeo-maven-plugin:1.0-SNAPSHOT:disassemble (bytecode-to-eo) on project jeo-it: Execution bytecode-to-eo of goal org.eolang:jeo-maven-plugin:1.0-SNAPSHOT:disassemble failed: java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "java.lang.CharSequence.toString()" because the return value of "org.cactoos.Scalar.value()" is null
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:333)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution bytecode-to-eo of goal org.eolang:jeo-maven-plugin:1.0-SNAPSHOT:disassemble failed: java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "java.lang.CharSequence.toString()" because the return value of "org.cactoos.Scalar.value()" is null
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:133)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "java.lang.CharSequence.toString()" because the return value of "org.cactoos.Scalar.value()" is null
    at jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance (DirectConstructorHandleAccessor.java:62)
    at java.lang.reflect.Constructor.newInstanceWithCaller (Constructor.java:502)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:486)
    at java.util.concurrent.ForkJoinTask.getThrowableException (ForkJoinTask.java:540)
    at java.util.concurrent.ForkJoinTask.reportException (ForkJoinTask.java:567)
    at java.util.concurrent.ForkJoinTask.invoke (ForkJoinTask.java:670)
    at java.util.stream.ForEachOps$ForEachOp.evaluateParallel (ForEachOps.java:160)
    at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateParallel (ForEachOps.java:174)
    at java.util.stream.AbstractPipeline.evaluate (AbstractPipeline.java:233)
    at java.util.stream.ReferencePipeline.forEach (ReferencePipeline.java:596)
    at org.eolang.jeo.Disassembler.disassemble (Disassembler.java:77)
    at org.eolang.jeo.DisassembleMojo.execute (DisassembleMojo.java:120)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
Caused by: java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "java.lang.CharSequence.toString()" because the return value of "org.cactoos.Scalar.value()" is null
    at org.cactoos.text.UncheckedText.lambda$new$0 (UncheckedText.java:52)
    at org.cactoos.func.UncheckedFunc.lambda$apply$0 (UncheckedFunc.java:37)
    at org.cactoos.scalar.Checked.value (Checked.java:56)
    at org.cactoos.scalar.IoChecked.value (IoChecked.java:44)
    at org.cactoos.scalar.Unchecked.value (Unchecked.java:37)
    at org.cactoos.func.UncheckedFunc.apply (UncheckedFunc.java:38)
    at org.cactoos.text.UncheckedText.asString (UncheckedText.java:76)
    at org.cactoos.text.TextOfScalar.toString (TextOfScalar.java:44)
    at org.cactoos.text.TextEnvelope.toString (TextEnvelope.java:37)
    at org.eolang.jeo.representation.License.value (License.java:43)
    at org.eolang.jeo.representation.License.toString (License.java:48)
    at org.xembly.Directives.set (Directives.java:400)
    at org.eolang.jeo.representation.directives.DirectivesProgram.iterator (DirectivesProgram.java:110)
    at org.xembly.Xembler.apply (Xembler.java:157)
    at org.xembly.Xembler.dom (Xembler.java:214)
    at org.xembly.Xembler.xml (Xembler.java:248)
    at org.xembly.Xembler.xmlQuietly (Xembler.java:225)
    at org.eolang.jeo.representation.MeasuredEo.<init> (MeasuredEo.java:30)
    at org.eolang.jeo.representation.BytecodeRepresentation.toEO (BytecodeRepresentation.java:98)
    at org.eolang.jeo.Disassembling.transform (Disassembling.java:67)
    at org.eolang.jeo.Caching.tryTransform (Caching.java:74)
    at org.eolang.jeo.Caching.transform (Caching.java:44)
    at org.eolang.jeo.Logging.transform (Logging.java:64)
    at org.eolang.jeo.Disassembler.disassemble (Disassembler.java:94)
    at org.eolang.jeo.ParallelTranslator.translate (ParallelTranslator.java:51)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.ArrayList$ArrayListSpliterator.forEachRemaining (ArrayList.java:1708)
    at java.util.stream.AbstractPipeline.copyInto (AbstractPipeline.java:509)
    at java.util.stream.ForEachOps$ForEachTask.compute (ForEachOps.java:291)
    at java.util.concurrent.CountedCompleter.exec (CountedCompleter.java:754)
    at java.util.concurrent.ForkJoinTask.doExec (ForkJoinTask.java:387)
    at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec (ForkJoinPool.java:1312)
    at java.util.concurrent.ForkJoinPool.scan (ForkJoinPool.java:1843)
    at java.util.concurrent.ForkJoinPool.runWorker (ForkJoinPool.java:1808)
    at java.util.concurrent.ForkJoinWorkerThread.run (ForkJoinWorkerThread.java:188)
Caused by: java.lang.NullPointerException: Cannot invoke "java.lang.CharSequence.toString()" because the return value of "org.cactoos.Scalar.value()" is null
    at org.cactoos.text.TextOfScalar.asString (TextOfScalar.java:39)
    at org.cactoos.text.UncheckedText.asString (UncheckedText.java:73)
    at org.cactoos.text.TextOfScalar.toString (TextOfScalar.java:44)
    at org.cactoos.text.TextEnvelope.toString (TextEnvelope.java:37)
    at org.eolang.jeo.representation.License.value (License.java:43)
    at org.eolang.jeo.representation.License.toString (License.java:48)
    at org.xembly.Directives.set (Directives.java:400)
    at org.eolang.jeo.representation.directives.DirectivesProgram.iterator (DirectivesProgram.java:110)
    at org.xembly.Xembler.apply (Xembler.java:157)
    at org.xembly.Xembler.dom (Xembler.java:214)
    at org.xembly.Xembler.xml (Xembler.java:248)
    at org.xembly.Xembler.xmlQuietly (Xembler.java:225)
    at org.eolang.jeo.representation.MeasuredEo.<init> (MeasuredEo.java:30)
    at org.eolang.jeo.representation.BytecodeRepresentation.toEO (BytecodeRepresentation.java:98)
    at org.eolang.jeo.Disassembling.transform (Disassembling.java:67)
    at org.eolang.jeo.Caching.tryTransform (Caching.java:74)
    at org.eolang.jeo.Caching.transform (Caching.java:44)
    at org.eolang.jeo.Logging.transform (Logging.java:64)
    at org.eolang.jeo.Disassembler.disassemble (Disassembler.java:94)
    at org.eolang.jeo.ParallelTranslator.translate (ParallelTranslator.java:51)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.ArrayList$ArrayListSpliterator.forEachRemaining (ArrayList.java:1708)
    at java.util.stream.AbstractPipeline.copyInto (AbstractPipeline.java:509)
    at java.util.stream.ForEachOps$ForEachTask.compute (ForEachOps.java:291)
    at java.util.concurrent.CountedCompleter.exec (CountedCompleter.java:754)
    at java.util.concurrent.ForkJoinTask.doExec (ForkJoinTask.java:387)
    at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec (ForkJoinPool.java:1312)
    at java.util.concurrent.ForkJoinPool.scan (ForkJoinPool.java:1843)
    at java.util.concurrent.ForkJoinPool.runWorker (ForkJoinPool.java:1808)
    at java.util.concurrent.ForkJoinWorkerThread.run (ForkJoinWorkerThread.java:188)
```

--- a/pom.xml
+++ b/pom.xml
@@ -237,6 +237,12 @@
       <version>1.37</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.yegor256</groupId>
+      <artifactId>together</artifactId>
+      <version>0.1.0</version>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
   <build>
     <testResources>

--- a/src/main/java/org/eolang/jeo/representation/License.java
+++ b/src/main/java/org/eolang/jeo/representation/License.java
@@ -8,6 +8,7 @@ import org.cactoos.Scalar;
 import org.cactoos.Text;
 import org.cactoos.io.ResourceOf;
 import org.cactoos.text.Sticky;
+import org.cactoos.text.Synced;
 import org.cactoos.text.TextOf;
 
 /**
@@ -35,7 +36,7 @@ public final class License implements Scalar<String> {
      * @param content The name of file with license.
      */
     public License(final String content) {
-        this.content = new Sticky(new TextOf(new ResourceOf(content)));
+        this.content = new Synced(new Sticky(new TextOf(new ResourceOf(content))));
     }
 
     @Override

--- a/src/test/java/org/eolang/jeo/representation/LicenseTest.java
+++ b/src/test/java/org/eolang/jeo/representation/LicenseTest.java
@@ -4,9 +4,12 @@
  */
 package org.eolang.jeo.representation;
 
+import com.yegor256.Together;
+import org.cactoos.list.ListOf;
 import org.cactoos.text.FormattedText;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
+import org.junit.jupiter.api.RepeatedTest;
 import org.junit.jupiter.api.Test;
 
 /**
@@ -20,12 +23,20 @@ final class LicenseTest {
     void readsLicenseFileCorrectly() throws Exception {
         final String name = "LICENSE.txt";
         MatcherAssert.assertThat(
-            new FormattedText(
-                "Unexpected file:'%s' content",
-                name
-            ).asString(),
+            new FormattedText("Unexpected file:'%s' content", name).asString(),
             new License(name).value(),
             Matchers.containsString("MIT")
+        );
+    }
+
+    @RepeatedTest(10)
+    void readsLicenseConcurrently() {
+        final License license = new License();
+        final int total = 10;
+        MatcherAssert.assertThat(
+            "Unexpected license content",
+            new ListOf<>(new Together<>(total, i -> license.value())),
+            Matchers.everyItem(Matchers.containsString("MIT"))
         );
     }
 }

